src/go/k8s/pkg/console/configmap.go

```diff
@@ -535,7 +535,7 @@ func (cm *ConfigMap) genKafka(username string) config.Kafka {
 			Enabled: y,
 			SchemaRegistry: config.ProtoSchemaRegistry{
 				Enabled:         y,
-				RefreshInterval: time.Second * 10,
+				RefreshInterval: time.Minute * 1,
```

Comment:
NIT: unrelated change as https://github.com/redpanda-data/redpanda-operator/pull/68 seems to address that.

Reply:
Ugh somehow I didn't branch off `main` 🤦 .